Libraries/Text/RCTTextFieldManager.m

```diff
@@ -75,7 +75,13 @@ - (BOOL)textFieldShouldEndEditing:(RCTTextField *)textField
 }
 
 RCT_EXPORT_VIEW_PROPERTY(caretHidden, BOOL)
-RCT_EXPORT_VIEW_PROPERTY(autoCorrect, BOOL)
+RCT_CUSTOM_VIEW_PROPERTY(autoCorrect, NSString, RCTTextField)
```

Comment:
nit: use correct type here UITextAutocorrectionType

Instead of repeating the logic here, extract this logic to an RCTConvert category.